ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config

```diff
@@ -0,0 +1,34 @@
+scripts=[
+"
+create path /content/dam(sling:Folder)
+create path /content/cq:tags(sling:Folder)
+
+create path /etc/packages(sling:Folder)
+create path /etc/tags(sling:Folder)
+
+create path /var/workflow(sling:Folder)
```

Comment:
This is already part of https://github.com/kwin/acs-aem-commons/blob/bugfix/install-var-package-in-aem-cloud/ui.apps/src/main/content/jcr_root/apps/acs-commons/config.author/org.apache.sling.jcr.repoinit.RepositoryInitializer-varworkflow-acls.config